docs.html

```diff
@@ -112,6 +116,101 @@ <h1>Alert</h1>
+                    <div class="badge badge-two-digit">
+                        <i class="fas fa-envelope"></i>
+                        <div class="badge-number"><span>99+</span></div>
+                    </div>
+                    <div class="badge badge-three-digit">
+                        <i class="fas fa-envelope"></i>
+                        <div class="badge-number"><span>999+</span></div>
+                    </div>
+                </div>
+              </div>
+              <iframe
+src="https://carbon.now.sh/embed/xpbmrkhok3eL3QSMPbAs"
+sandbox="allow-scripts allow-same-origin">
+</iframe>
+              <h2>Badge on avatars</h2>
+              <div class="example-container">
+                <div class="container">
+                    <div class="badge-avatar">
+                        <img src="./images/mhd-dp.jpeg" alt="avatar" class="avatar avatar-md">
+                        <div class="badge-status badge-online"></div>
+                    </div>
+                    <div class="badge-avatar">
+                        <img src="./images/mhd-dp.jpeg" alt="avatar" class="avatar avatar-md">
+                        <div class="badge-status badge-busy"></div>
+                    </div>
+                </div>
+              </div>
+              <iframe
+src="https://carbon.now.sh/embed/drCeQikuvuqoEPveQyIu"
+sandbox="allow-scripts allow-same-origin">
+</iframe>
+          </section>
+          <section id="button">
+            <h1>Button</h1>
+            <p>
+              Buttons are the elements that call user to actions. <br>
+              Primary button allow user to accomplish the common or important action. <br>
+              Secondary button indicates of action that are of less importance
+            </p>
+            <div class="example-container">
+              <button class="btn btn-primary">Primary</button>
+              <button class="btn btn-secondary">Secondary</button>
+              <button class="btn btn-disabled" disabled>Disabled</button>
+            </div>
+            <br>
+            <iframe
+src="https://carbon.now.sh/embed/WbxfufpLNudamdULsbkX"
+sandbox="allow-scripts allow-same-origin">
+</iframe>
+            <h2>Link button</h2>
+            <div class="example-container"><button class="btn btn-link">Link</button></div>
+            <br>
+            <iframe
+src="https://carbon.now.sh/embed/bY3Q8AatlMqzOSmKl5BU"
+sandbox="allow-scripts allow-same-origin">
+</iframe>
+            <h2>Icon button</h2>
+            <p>
+              Icon button has icons associated with corresponding call to action.
+            </p>
+            <div class="example-container">
+              <button class="btn-icon"><i class="fas fa-trash"></i></button>
+              <button class="btn-icon"><i class="fas fa-camera"></i></button>
+            </div>
+            <br>
+            <iframe
+src="https://carbon.now.sh/embed/LLQJxun9Z7nqYWZrufZY"
+sandbox="allow-scripts allow-same-origin">
+</iframe>
```

Comment:
**Suggestion**: The code indentation looks a bit messed up here. Maybe you can correct this up.

Reply:
sure vishal.thank you